talent/sourcerers/README.md

```diff
@@ -0,0 +1,11 @@
+## Questions to learn sourcerers better
+
+According to research, it becomes harder to establish a personal relationship with other team members when working remotely.
+
+In order to know you better, as a new member we would like you to reply a few have personal questions, as well as share with us your social media accounts you are comfortable with. By doing so we can establish more personal relationships outside of work activities.
+
+We have created a list of the questions and below are the instructions on how to successfully respond to the questions and make other members read them.
```

Comment:
should be answered **all** the questions?
I think if someone does not want to answer some of them, she/he should feel free to remove it from its questionnaire.

If so, I'd explain it in this README.